resources/charts/kubernetes/templates/kubelet-ds.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet
+  namespace: kube-system
+  labels:
+    tier: node
+    k8s-app: kubelet
+spec:
+  selector:
+    matchLabels:
+      tier: node
+      k8s-app: kubelet
+  template:
+    metadata:
+      labels:
+        tier: node
+        k8s-app: kubelet
+    spec:
+      containers:
+      - name: kubelet
+        image: {{ .Values.kubelet.image }}
+        command:
+        - ./hyperkube
+        - kubelet
+        - --node-ip=$(NODE_PRIVATE_IP)
+        - --anonymous-auth=false
+        - --authentication-token-webhook
+        - --authorization-mode=Webhook
+        - --client-ca-file=/etc/kubernetes/ca.crt
+        - --cluster_dns={{ .Values.kubelet.clusterDNS }}
+        - --cluster_domain={{ .Values.kubelet.clusterDomain }}
+        - --cni-conf-dir=/etc/kubernetes/cni/net.d
+        - --config=/etc/kubernetes/kubelet.config
+        - --kubeconfig=/etc/kubernetes/kubeconfig
+        - --lock-file=/var/run/lock/kubelet.lock
+        - --network-plugin=cni
+        - --node-labels=node.kubernetes.io/node
+        - --pod-manifest-path=/etc/kubernetes/manifests
+        - --read-only-port=0
+        - --volume-plugin-dir=/var/lib/kubelet/volumeplugins
```

Comment:
The `taints` and `labels` field is removed here. Because this is one config for all the kubelets in the cluster.

Reply:
This `kubelet` should never "register" itself, so IMO it should be fine to skip those flags here.

Reply:
Then who should do it?

Reply:
We need to have that logic somewhere?

Reply:
As in the main thread comment:
>I initially though, that self-hosted kubelet will never perform registering into the cluster, but now that I think about it, If one removes Node object manually, it will re-register then, so we shouldn't really remove taints/labels flags.

That means, we should keep `node-labels` and `taints` flags.

Reply:
I think this chart should reside at the lokomotive-kubernetes level. Because there we have pool specific information of taints and labels? WDYT?

Reply:
Well, that's problematic unfortunately. Because lokomotive-kubernetes only applies the charts during bootstrapping. Also controller pool has no knowledge about worker pools...

Reply:
That's solved now.